src/app/services/actions/permissions-action-creator.ts

```diff
@@ -9,47 +13,60 @@ export function fetchScopesSuccess(response: object): IAction {
   };
 }
 
+export function fetchScopesPending(): any {
```

Comment:
>any [](start = 38, length = 3)

i know you don't like creating an interface for something this simple but it really does help keep the code more readable and maintainable.